### ISSUE TITLE
Determine how best to handle jQuery dependency

### ISSUE THREAD

**Issue by fureigh** [18F/web-design-standards-drupal-d7#2]
As of this writing, Drupal 8 includes jQuery v1.7. The current version of the USWDS uses jQuery v1.11.3.

**Comment by jonraedeke** [comment]
I'm thinking the best short term approach is to add documentation to the project page like [Bootstrap](https://www.drupal.org/project/bootstrap). Forcing the upgrade by loading it as an external library would break some core and contrib js.

If we added it with theme, we should make a theme setting to opt out in case a contrib module is being use manage the upgrade.

**Comment by AlexBrown-appnovation** [comment]
I agree with @jonraedeke about including this in documentation. 

However I like the way that the Aurora theme has decided to tackle this with their owned hosted github page with a list of suggested modules - one of which is jquery_update. In our case, maybe we have a section of the documentation called 'Required Modules'. 

Aurora documentation: http://snugug.github.io/Aurora/

**Comment by fureigh** [comment]
Whoops, thought I'd replied to this earlier. Thanks very much for the examples!

For the Drupal 7 version, I like the idea of including jQuery v1.11.3 as a requirement in the documentation. Giving users the option to opt out would require also testing the USWDS against earlier versions of jQuery, which could get very brittle very quickly.

For Drupal 8,[ jQuery update has been added to core](https://www.drupal.org/node/2607768):

> D8 ships with the latest, and the plan is to continue increasing the shipped version (with BC available) in further minor releases.

We'll therefore want to keep an eye on how closely new releases of the USWDS and D8 move in tandem (or don't).

**Comment by kevwalsh** [comment]
There's a [jquery_update patch for v1.11.4](https://www.drupal.org/project/issues/jquery_update?text=&status=All&priorities=All&categories=All&version=All&component=jQuery+1.11.x), but we may need to defer this for d7 until it's in a stable release.

**Comment by fureigh** [comment]
:+1: Thanks for finding that, @kevwalsh!

**Comment by brockfanning** [comment]
I don't think this is an issue anymore, as I believe jquery is no longer a dependency in USWDS.
